CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ Release report: TBD
 
 ### Changed
 
+- Refactor FIM test `test_remove_audit` to retry to execute a command if fail ([#3562](https://github.com/wazuh/wazuh-qa/pull/3562)) \- (Tests)
```

Comment:
Too vague, what command?
Also, I should say that it is more of a fix than a refactor.

Reply:
Done [here](https://github.com/wazuh/wazuh-qa/pull/3562/commits/f458d8076aecd468d05c996358a5bc702213548a)